modules/nw-sriov-add-pod-runtimeconfig.adoc

```diff
@@ -99,3 +99,103 @@ $ oc create -f <filename> <1>
 ----
 $ oc exec sample-pod -n <namespace> -- ip addr show
 ----
+
+= Configuring static GUID and IP addresses on additional SR-IOV InfiniBand networks
+
+You can configure static Global Unique Identifiers (GUID) and IP addresses on an SR-IOV InfiniBand network by specifying Container Network Interface (CNI) runtimeConfig data in a Pod annotation.
+
+.Prerequisites
+
+* Install the {product-title} Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges when creating the SriovNetwork CR.
+
+.Procedure
+
+. Create the following SriovIBNetwork CR, and then save the YAML in the `<name>-sriov-ib-network.yaml` file. Replace `<name>` with a name for this additional network.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovIBNetwork
+metadata:
+  name: <name> <1>
+  namespace: openshift-sriov-network-operator <2>
+spec:
+  networkNamespace: <target_namespace> <3>
+  ipam: '{ "type": "static" }' <4>
+  capabilities: '{ "infiniband_guid": true, "ips": true }' <5>
```

Comment:
`keyword` should be `"infinibandGUID": true`

Reply:
Done